extensions/EXT/GLX_EXT_no_config_context.txt

```diff
@@ -0,0 +1,254 @@
+
+Issues
+
+ 1) Should non-conformant no-config contexts be allowed to be created?
+
+    RESOLVED: No. We are not encouraging non-conformant contexts.
+
+ 2) Are no-config contexts constrained to those GL & ES implementations
+    which can support them?
+
+    RESOLVED: Yes. ES2 + OES_surfaceless_context, ES 3.0, and GL 3.0 all
+    support binding a surface without a context. This implies that they
+    don't need to know surface attributes at context creation time.
+
+ 3) For an OpenGL or OpenGL ES context created with no config, what is the
+    initial state of GL_DRAW_BUFFER and GL_READ_BUFFER for the default
+    framebuffer?
+
+    RESOLVED: This is an implementation detail rather than a spec issue.
+    glReadBuffer/glDrawBuffer have undefined results if called without a
+    current context. The GL_DRAW_BUFFER and GL_READ_BUFFER are set on the
+    first glXMakeCurrent call and can be updated in glReadBuffer and
+    glDrawBuffers calls after that. Therefore, the attribute value with
+    which the context is created is irrelevant from the point of view of the
+    spec and is left up to the implementation.
+
+ 4) Can glXMakeCurrent alter the GL_DRAW_BUFFER and GL_READ_BUFFER state of
+    the default framebuffer?
+
+    RESOLVED: Yes, but only on the first call to glXMakeCurrent. The two
+    relevant excerpts from the OpenGL 3.2 Core Profile Specification.
+    From Section 4.2.1 Selecting a Buffer for Writing:
+
+        For the default framebuffer, in the initial state the draw buffer
+        for fragment color zero is BACK if there is a back buffer; FRONT if
+        there is no back buffer; and NONE if no default framebuffer is
+        associated with the context.
+
+    From 4.3.3 Pixel Draw/Read State:
+
+        For the default framebuffer, in the initial state the read buffer is
+        BACK if there is a back buffer; FRONT if there is no back buffer;
+        and NONE if no default framebuffer is associated with the context.
+
+    Based on the above excerpts on the first call to glXMakeCurrent the
+    GL_DRAW_BUFFER and GL_READ_BUFFER are set to: GL_NONE if the surface is
+    NULL, GL_BACK if the surface is double buffered, GL_FRONT if the surface
+    is single buffered. Following calls to glReadBuffer and glDrawBuffers
+    change the GL_DRAW_BUFFER and GL_READ_BUFFER attributes and these values
+    persist even when the application change the current context.
+
+ 5) Should we add an glXCreateGenericContext which is the same as
+    glXCreateContext but without the config parameter?
+
+    RESOLVED: No.
+
+ 6) Can no-config contexts share state with contexts that has a config?
+
+    RESOLVED: Yes. This extension implies that the dependency of the context
+    on the config is quite minimal so no restriction w.r.t sharing should be
+    enforced.
+
+ 7) What surface types can be made current with a no-config context?
+
+    RESOLVED: any surface type supported by the implementation can be made
```

Comment:
I'm assuming by supported, you mean which can be created according to the FBConfigs it exports, not that the implementation can arbitrarily reject random drawables it doesn't like here.  Also, s/surface type/GLXDrawable types/.  Surface is an EGLism.